torch_xla/csrc/ops/dynamic_ir.h

```diff
@@ -74,4 +75,18 @@ class SizeDiv : public XlaNode, public torch::lazy::DimensionNode {
   virtual XlaOpVector Lower(LoweringContext* loctx) const override;
 };
 
+class SizeConstant : public torch_xla::Scalar,
```

Comment:
Can you point me to the spec of this class? I was looking at the design doc "Bounded Dynamic Shape Design for PyTorch" by Milad but didn't find it.

Reply:
it is new.. well, it just represent a constant, so you can do something like `a.size + 5` where a is a tensor. You need a ir representation of this computation.